### ISSUE TITLE
:bug:[spacy 3.0.0rc2]: spacy info exits with error using model path

### ISSUE THREAD

**Issue by AnotherStranger** [explosion/spaCy#6650]
## How to reproduce the behaviour

Running spacy info on a model produces an Error.
I included the error below:
```
Traceback (most recent call last):
  File "<virtenv_path>/bin/spacy", line 8, in <module>
    sys.exit(app())
  File "<virtenv_path>/lib/python3.8/site-packages/typer/main.py", line 214, in __call__
    return get_command(self)(*args, **kwargs)
  File "<virtenv_path>/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "<virtenv_path>/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "<virtenv_path>/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "<virtenv_path>/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "<virtenv_path>/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "<virtenv_path>/lib/python3.8/site-packages/typer/main.py", line 497, in wrapper
    return callback(**use_params)  # type: ignore
  File "<virtenv_path>/lib64/python3.8/site-packages/spacy/cli/info.py", line 27, in info_cli
    info(model, markdown=markdown, silent=silent)
  File "<virtenv_path>/lib64/python3.8/site-packages/spacy/cli/info.py", line 36, in info
    data = info_model(model, silent=silent)
  File "<virtenv_path>/lib64/python3.8/site-packages/spacy/cli/info.py", line 86, in info_model
    meta_path = model_path / "meta.json"
TypeError: unsupported operand type(s) for /: 'str' and 'str'
```
Following command produced the error: 
```
spacy info <path_to_model>
```

## Your Environment

* Operating System: Fedora 33
* Environment Information: everything is installed within a poetry environment
## Info about spaCy

- **spaCy version:** 3.0.0rc2
- **Platform:** Linux-5.9.16-200.fc33.x86_64-x86_64-with-glibc2.2.5
- **Python version:** 3.8.7
- **Pipelines:** de_core_news_lg (3.0.0a0)

I hope that i included every relevant information to reproduce the Error.

**Comment by svlandeg** [comment]
Thanks, I can reproduce this, we'll look into it!

**Comment by AnotherStranger** [comment]
> Thanks, I can reproduce this, we'll look into it!

That reaction was fast...
Thanks!

**Comment by svlandeg** [comment]
Fixed by PR #6651 :-)
